### DISCUSSION TITLE
CC65 flagged as a potential threat ? VirusTotal

### DISCUSSION THREAD

**Discussion by Oxey405** [cc65/cc65#2228] [General]
Hello to the CC65 contributors !
I was looking into tinkering a bit with cc65 whilst following the tutorial of "NesHacker" on youtube
For safety I do a check of the file downloaded from SourceForge. The result (you can find on the URL below) tells me Google and 2 others (a bit unknown) antivirus companies flagged the file as a potential threat / trojan.
I would like to know if you are aware of this and for my safety I will not extract the file until I get proof that CC65 is safe.
Until then, have a great day.
The link to the scan :
https://www.virustotal.com/gui/file/a2d867346f21d836780e1cbc694d69b646a27bd349ec1e987e5ceef7accc4ec7

**Comment by mrdudz** [comment]
Am Dienstag, 27. Juni 2023, 01:34:43 CEST schrieb Oxey405:
> Hello to the CC65 contributors !
> I was looking into tinkering a bit with cc65 whilst following the tutorial
> of "NesHacker" on youtube For safety I do a check of the file downloaded
> from SourceForge. The result (you can find on the URL below) tells me
> Google and 2 others (a bit unknown) antivirus companies flagged the file as
> a potential threat / trojan. I would like to know if you are aware of this
> and for my safety I will not extract the file until I get proof that CC65
> is safe. Until then, have a great day.
> The link to the scan :
> https://www.virustotal.com/gui/file/a2d867346f21d836780e1cbc694d69b646a27bd3
> 49ec1e987e5ceef7accc4ec7

While virustotal is a nice tool - you have to know how to interpret the
results... especially when its just a few, and unknown, engines that complain,
its most likely a false positive. It happens frequently :)

--

http://hitmen.eu                 http://ar.pokefinder.org
http://vice-emu.sourceforge.net  http://magicdisk.untergrund.net

It is not possible to simultaneously understand and appreciate the Intel
architecture.
<Ben Scott>

**Comment by groessler** [comment]
And if you want to be sure that you don't get a virus, build from source...

**Comment by bbbradsmith** [comment]
This should be filed as an issue with VirusTotal not with cc65.

**Comment by mrdudz** [comment]
> This should be filed as an issue with VirusTotal not with cc65.

Not even that - virustotal is just a service that runs a dozen or so different engines - you'll have to contact the vendor of the respective engine.

However, with something like the automatic builds of cc65 that is a futile exercise, as the binaries change often. There is also nothing we can do about it. The only way to be 100% sure is to build from source yourself.

(moving to discussions)
